pkg/network/tracer/perf_batching.go

```diff
@@ -106,7 +106,7 @@ func (p *PerfBatchManager) GetIdleConns() []network.ConnectionStats {
 			start = bState.offset
 		}
 
-		idle = p.extractBatchInto(idle, b, start, batchLen)
+		idle = append(idle, p.extractBatchInto(idle, b, start, batchLen)...)
```

Comment:
`idle` (`buffer` in `extractBatchInto`) is appended to internally, and then returned.

Reply:
Yes, back to the drawing board I guess.